### ISSUE TITLE
Closed

### ISSUE THREAD

**Issue by ahmadawais** [liviuschera/pure-code#2]

**Comment by ahmadawais** [comment]
From the comments in your theme's file, it is pretty evident that you started with by forking [Shades of Purple](https://marketplace.visualstudio.com/items?itemName=ahmadawais.shades-of-purple). Then modified the colors and arrangement a bit. Also your theme is not arranged by the way ref is present.


### Theme Reference Proof!

On theme ref the title bar and welcome page color ref is placed in different places. But I thought that it should be combined for a reason. So, I combined it inside my theme.

![image](https://on.ahmda.ws/rMM0/c)

Here's what you did in your forked theme

![image](https://on.ahmda.ws/rMq9)

You copied the same title bar and welcome bar colors, you even copied a few of my hex colors and changed theme a tiny bit, you see how I know? Coz I wrote `#A599E9` capital everywhere. You followed the suit but other colors in your theme just like mine are not always capital.

Yes, you changed your theme a fair bit but you just pretty much forked mine and changed the hex colors with minor modification.

And guess what, I am all OK with it. But not OK with you not giving the fair credit where it's due. And then commenting like `What's your angle here buddy?`. Do you seriously think I wouldn't notice? This is pathetic.

I've lost all hope on you!

**Comment by ahmadawais** [comment]
Oh so sad to see that. Even with so much proof you fail to accept that you forked the theme. I mentioned it several times that I am OK with your fork. And don't even require attribution but the fact that you're claiming to not have forked the theme is beyond me. It's like lying about donating to Charity or something. 

And sure enough, I have nothing to hide. I forked the theme I used and praised since I couldn't stand an online bully. Ask me anything about it. 

Since you don't want to even accept that you forked my work, I don't see the reason for me to waste more time commenting here. 

And yes it was wishful thinking tthat you will accept what you have done (which would have been) completely fine with me. But your poor handling of this subject speaks a lot about you.
